### ISSUE TITLE
Duplicates during sorts

### ISSUE THREAD

**Issue by Brand3000** [outl1ne/nova-sortable#192]
Hi! Watch the video [here](https://brand.dn.ua/nova/sorting.mov) please.

**Comment by LuckyLukas98** [comment]
Probably the same issue as #184 ?

**Comment by Brand3000** [comment]
Yes, it's the same issue, but that fix doesn't help. It fixes duplicates but the sorting process doesn't work properly either.

**Comment by Brand3000** [comment]
If we leave the :key attribute without id, the issue disappears. Actually, I think there's an issue with the vue-draggable-next package. I have written about this issue to their thread of issues.

**Comment by leonardyrj** [comment]
up

**Comment by Tarpsvo** [comment]
Heya! Version 3.4.7 should now address this issue. Please give it a test run and let me know if it solves the issue. Thanks!

**Comment by Brand3000** [comment]
Nope. It sorts and duplicates don't appear. However it leaves previous visual order of items.

**Comment by Tarpsvo** [comment]
Yeah, I manage to reproduce this as well now. I think it might be a fault of vue-draggable. I can't seem to understand why it would do that. It works in 95% of cases, but sometimes it refuses to rearrange some combination of rows. However it works when you do the inverse order. Ie row 1->3 does not work, but row 3->1 works.

**Comment by Brand3000** [comment]
Yep, I agree and I've already written the issue to that package developers. It seems that they don't care though =(
